winton_kafka_streams/kafka_streams.py

```diff
@@ -7,9 +7,9 @@
 import threading
 from enum import Enum
 
-from .processor import StreamThread
+from winton_kafka_streams.errors.kafka_streams_error import KafkaStreamsError
```

Comment:
Any reason not to use relative imports like elsewhere (here and a couple of other modules too)?

Reply:
I thought I'd corrected these. Evidently I missed at least one. I think this is auto-generated by PyCharm.